UI/ui_mainmode.csv

```diff
@@ -3500,13 +3500,13 @@ FilterLaveCave#0,"""$(0). Volcan"""
 FilterDesert#0,"""$(0). Desierto"""
 FilterSnowMountain#0,"""$(0)．凍土"""
 FilterTunnel#0,"""$(0). Tuneles"""
-FilterFly#0,"""$(0). Continente flotante"""
+FilterFly#0,"""$(0). Skyscape"""
```

Comment:
WHY?

Reply:
Continente flotante is too long... i'll fix that

Reply:
https://i.imgur.com/9sOI4XM.png